dist/pom.xml

```diff
@@ -413,6 +424,14 @@
               <id>restore</id>
               <mainClass>io.camunda.zeebe.restore.RestoreApp</mainClass>
             </program>
+            <program>
+              <id>operate</id>
+              <mainClass>io.camunda.operate.Application</mainClass>
```

Comment:
❓ This application class is still living in the webapp module that is correct yes?

Reply:
Exactly, while I have done this differently in the POC, I decided here to keep the `Application` where it is right now. For achieving the Single Jar/Distribution this is sufficient.